en-US/win10/samples/NodejsWULightning.md

```diff
@@ -76,12 +77,12 @@ Here's what the code above is doing:
 * Every 1000 milliseconds (1 second), the value of the LED is checked and then set to the opposite of the current value.
 
 ### Add Microsoft.IoT.Lightning.Providers library to your project
-* Clone the bus provider repository on your PC with `git clone https://github.com/ms-iot/BusProviders`.
-* In the Solution Explorer in your project, right click on the solution node, select 'Add->Existing Project...', then add \<Bus Provider Clone Root\>\Microsoft.IoT.Lightning.Providers\Providers\Microsoft.Iot.Lightning.Providers.vcxproj.
-* Build the solution.
-* In the Solution Explorer, right click on your Node.js project (with server.js), select 'Add->Existing Item...'. then add the following files:
-  * \<Project Root\>\ARM\Debug\Microsoft.Iot.Lightning.Providers\Microsoft.IoT.Lightning.Providers.dll
-  * \<Project Root\>\ARM\Debug\Microsoft.Iot.Lightning.Providers\Microsoft.IoT.Lightning.Providers.winmd
+* In the Solution Explorer in your project, right click on the project node (with server.js), select 'Open Command Prompt Here...'
+* Run `nuget install Microsoft.IoT.Lightning -Pre`
+* In the Solution Explorer, right click on your Node.js project again, select 'Add->Existing Item...'. then add the following files:
+  * \<Project Root\>\Microsoft.IoT.Lightning.1.1.0\lib\uap10.0\Microsoft.IoT.Lightning.Providers.winmd
+  * \<Project Root\>\Microsoft.IoT.Lightning.1.1.0\runtimes\win10-\<arm|x86|x64\>\native\Microsoft.IoT.Lightning.Providers.dll
+  * \<Project Root\>\Microsoft.IoT.Lightning.1.1.0\runtimes\win10-\<arm|x86|x64\>\native\Lightning.dll
```

Comment:
Starting with v1.1.0, Microsoft.IoT.Lightning.Providers.dll doesn't depend on the Lightning DLL anymore, so let's remove the line with Lightning.dll.